index.html

```diff
@@ -42,5 +43,224 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
       </ul>
     </div>
   </section>
+
+<!-- Section: Recent Works -->
+<section class="RecentWorks">
+
+  <div class="RecentWorksSection">
+    <h3 class="Title_MyRecentWorks">My recent Works</h3>
+  </div>
+  
+  <!-- ************ grid used to display diferent project cards ************ -->
+
+  <div class="grid_works">   
+
+    <!-- ************* Main card ************* -->
+    
+    <div class="main_card_cnt">
+      <div class="cnt_img_main_card">
+        <img class="main_card_cnt_img" src="/images/card1_img.png" alt="Project image">
+      </div>
+      <div class="cnt_info_main_card">
+        <h3 class="MainCardTitle Title2_dark">Multi-Post Stories</h3>
+        <p class="MainCardDescrip Simpletext">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
I realize that the font family on this and the details of the rest of the projects in your works section in ```poppins``` in your design whereas it is ```Inter``` in the figma design. I recommend you change this to make your design identical to the figma design 😉

Reply:
The focus of my comment is on the p tag on line 66 👍